app/services/maat_link_creator.rb

```diff
@@ -6,31 +6,41 @@ def initialize(laa_reference_id:)
   end
 
   def call
-    push_to_sqs unless laa_reference.dummy_maat_reference?
-    call_common_platform_endpoint
-    PastHearingsFetcherWorker.perform_at(30.seconds.from_now, Current.request_id, prosecution_case_id)
+    publish_laa_reference_to_queue
+    post_laa_references_to_common_platform
+    fetch_past_hearings
   end
 
 private
 
-  def push_to_sqs
+  def publish_laa_reference_to_queue
+    return if laa_reference.dummy_maat_reference?
+
     Sqs::PublishLaaReference.call(defendant_id: laa_reference.defendant_id,
```

Comment:
Any thoughts on renaming the Sqs module, so we're not thinking in terms of AWS in case we use a different queue service in the future? Different question for another time, but just a thought

Reply:
Totally agree, our code should not change if we decided to switch queueing providers 👌🏻